promscale/installation/rhel-centos.md

```diff
@@ -0,0 +1,38 @@
+# Install Promscale on Redhat or CentOS
+You can install Promscale on Redhat or CentOS system where you have an
+already installed and working Prometheus or OpenTelemetry environment.
+
+## Install TimescaleDB
+To use with Promscale you need TimescaleDB database, as well as the Promscale
+extension. The Promscale extension contains support functions to improve
+performance of Promscale.
+<procedure>
+
+### Installing TimescaleDB and Promscale extension
+
+1.  Install TimescaleDB following the instructions in the
+    [TimescaleDB install page][tsdb-install-self-hosted]
+1.  Install Promscale extension
+    ```
+    yum install -y promscale-extension-postgresql-14
```

Comment:
It's not very good practice to tell people to override safety questions in command prompts, even if we all know we do it 😉 

```suggestion
    yum install promscale-extension-postgresql-14
```